distribution/docs/src/main/resources/_contents/_resources/content-directory-monitor-contents.adoc

```diff
@@ -23,93 +21,55 @@ include::{adoc-include}/_tables/org.codice.ddf.catalog.content.monitor.ContentDi
-** The Catalog Framework generates a GUID for the create request for this ingested file.
-** The Catalog Framework then invokes the Catalog Content Plugin, which looks up the Input Transformer associated with the ingested file's mime type and invokes the Catalog Framework, which inserts the metacard into the catalog. This Input Transformer creates a metacard based on the contents of the ingested file.
-** The Catalog Framework sends back a successful status to the Camel route that was monitoring the directory.
-** The Camel route completes and moves the file from the monitored directory to its `.ingested` sub-directory.
-
-.Imported Services
-[cols="3*", options="header"]
-|===
+The Content Directory Monitor processes files in a directory, and all of its sub-directories. The Content Directory Monitor offers three options:
 
-|Registered Interface
-|Availability
-|Multiple
+* Delete
+* Move
+* Monitor in place
 
-|`ddf.mime.MimeTypeToTransformerMapper`
-|required
-|false
+Regardless of the option, the ${branding} takes each file in a monitored directory structure and creates a metacard for it. The metacard is linked to the file. The behavior of each option is given below.
 
-|`ddf.catalog.CatalogFramework`
-|required
-|false
+* *Delete*
+** Copies the file into the Content Repository.
+** Creates a metacard in the Catalog from the file.
+** ERASES THE ORIGINAL FILE FROM THE MONITORED DIRECTORY.
 
-|`ddf.catalog.filter.FilterBuilder`
-|required
-|false
 
-|===
+* *Move*
+** *Copies the file into the directory `.\ingested`*
+** Copies the file into the Content Repository.
+** Creates a metacard in the Catalog from the file.
+** ERASES THE ORIGINAL FILE FROM THE MONITORED DIRECTORY.
 
-.Exported Services
-[cols="3*", options="header"]
-|===
+* *Monitor in place*
+** Creates a metacard in the Catalog from the file.
+** Creates a reference from the metacard to the original file in the monitored directory.
+** If the original file is deleted, the metacard is removed from the Catalog.
+** If the original file is changed the metacard is updated to reflect the new content. The old file is removed from the Content Repository and the updated file is added to the Content Repository.
+
+.Atribute Mapper
+The Content Directory Monitor supports setting metacard attributes directly when ${branding} ingests a file. Custom mappings are entered in the form:
+
+`*attribute-name=attribute-value*`
+
+For example, to set the contact email for all metacards, add the custom mapping:
+
+`*contact.point-of-contact-email=doctor@clinic.com*`
+
+Each mapping sets the value of a single metacard attribute. To set the value of an additional attribute, select the "plus"
+ icon in the UI. This creates an empty line for the entry.
+
+To set multi-valued attributes, use a comma separated list for each value. For example, to add the keywords _PPI_ and _radiology_ to each metacard, add the custom attribute mapping:
+
+`*topic.keyword=PPI,radiology*`
```

Comment:
Now that I look at what I wrote this might not currently be correct...looks like I implemented support for `foo=baz,foo=bar` -> `foo=bar,baz`, I think this was due to camel limitations. I can either update the documentation or try and come up with a different implementation.